que_test.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from que_ import Queue
+
+
+def test_innit():
+    my_queue = Queue([1, 2, 3])
+    assert my_queue.dll.tail.value == 1
+    assert my_queue.dll.tail.prev.value == 2
+    assert my_queue.dll.head.value == 3
+    assert my_queue.dll._size == 3
+
+
+def test_enqueue():
+    my_queue = Queue()
+    my_queue.enqueue(1)
+    assert my_queue.dll.tail.value == 1
+    assert my_queue.dll.head.value == 1
```

Comment:
You testing that the doubly linked list looks a certain way. When possible only test the code that directly used. 

In this case is enqueue(1), enqueue(2),  enqueue(3), and then dequeue(), dequeue(), dequeue() returns what you expect, then you know it is working.  It doesn't matter how the dll works on the inside. You've already written tests that prove it is working as you expect.